Tools/ProviderOptionContract.py

```diff
@@ -46,4 +49,9 @@ def IsTradable(self):
     def OpenInterest(self):
         return self.security.OpenInterest
 
+    @property
+    def implied_volatility(self):
+        return self.context.iv(self.symbol, self.mirror_symbol, resolution=Resolution.HOUR)
```

Comment:
where is self.context.iv from? 
It does not seem like the IV indicator is assigned to the context anywhere.